omnibus/config/software/init-scripts-agent.rb

```diff
@@ -0,0 +1,110 @@
+name 'init-scripts-agent'
+
+description "Generate and configure init scripts packaging"
+
+always_build true
+
+build do
+  if linux_target?
+    etc_dir = "/etc/datadog-agent"
+    mkdir "/etc/init"
+    if debian_target?
+      # sysvinit support for debian only for now
+      mkdir "/etc/init.d"
+
+      # debian recommends using a different directory for systemd unit files
+      systemd_directory = "/lib/systemd/system"
+      erb source: "upstart_debian.conf.erb",
+          dest: "/etc/init/datadog-agent.conf",
+          mode: 0644,
```

Comment:
Just to confirm, we're basically merging two existing steps into one by directly generating the files in their final, location, right?